content/en/blog/releases/release_v0.6.0.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Announcing OpenFunction 0.6.0: FaaS observability, HTTP trigger, and more"
+linkTitle: "Release v0.6.0"
+date: 2022-03-25
+weight: 95
+
+---
+
+[OpenFunction](https://github.com/OpenFunction/OpenFunction) is a cloud-native open source FaaS platform aiming to let you focus on your business logic. The OpenFunction community has been hard at work over the past several months preparing for the v0.6.0 release. Today, we proudly announce v0.6.0 is officially available. Thanks to the community for helping drive the new features, enhancements, and bug fixes.
+
+OpenFunction 0.6.0 brings notable features including function plugin, distributed tracing for functions, control autoscaling behavior, HTTP trigger to async function, etc. Meanwhile, OpenFuncAsync runtime definition has also been refactored. The core API has been upgraded from v1beta1 to v1alpha1. 
```

Comment:
I have some suggestions:
0.6.0 -> v0.6.0
OpenFuncAsync -> Async
from v1beta1 to v1alpha1 -> from v1alpha2 to v1beta1

Reply:
agree

Reply:
Thanks. There is only one point I disagree. Generally, we can say `OpenFunction 0.6.0` or only `v0.6.0`. This is quite commonly used in some other places such as [Kubernetes 1.23](https://kubernetes.io/blog/2021/12/07/kubernetes-1-23-release-announcement/).

Reply:
there're also cases using v1.x.x https://blog.dapr.io/posts/2021/02/17/announcing-dapr-v1.0/
Both are ok